buildbox/scripts/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get dist-upgrade
```

Comment:
Also need a `sudo apt-get upgrade` 

If we are doing a `dist-upgrade` we will need to restart the box in a post-install.sh